src/chapters/setup-intro-data-types/some-java-practice.rst

```diff
@@ -10,27 +10,16 @@ Celsius.
 Temperature Conversion
 -----------------------
 
-If you need to review the steps for creating a new IntelliJ project, refer
-back to the :ref:`First Java Program <create-new-java-project>` instructions.
+#. Open the ``TempConverter`` file in the ``java-web-dev-exercises`` project in IntelliJ.
```

Comment:
How about adding a screen shot showing the file tree IJ?

Reply:
added, although fyi all of these are subject to change since the repo will be changing as we add items to it